packages/frontend-shared/cypress/support/onlineNetwork.ts

```diff
@@ -0,0 +1,62 @@
+export function goOnline () {
```

Comment:
This turned out nice @estrada9166

Reply:
Yeah! it was pretty cool to have it working!